readme-asana.md

```diff
@@ -0,0 +1,67 @@
+# [Taskwarrior](https://taskwarrior.org/) ⬄ [Asana](https://www.asana.com)
+
+## Description
+
+Synchronize Asana tasks.
+
+Upon execution, `tw_asana_sync` will synchronize, and on subsequent runs of the
+program keep synchronized, the following attributes:
+
+- Asana task `name` <-> TW task `description`
+- Asana task `completed` <-> TW task `status`
+- Asana task `due_at` or `due_on` <-> TW task `due`
+- Asana task is deleted <-> TW task is deleted
+
+Current limitations:
+
+- Does not sync Asana tags, project names, subtasks, projects, likes, etc.
+- Does not sync Taskwarrior tags, projects, annotations, etc.
+- Only supports authentication with an Asana [Personal Access Token](https://developers.asana.com/docs/personal-access-token)
+- Only syncs Asana tasks that are assigned to the user associated with that access token
+
+## Usage
+
+### Find IDs of available workspace
+
+```sh
+$ tw_asana_sync --list-asana-workspaces
+
+Asana workspaces:
+====================
+
+- My Workspace: gid=123456789012345
+```
+
+### Synchronize all workspace tasks
+
+To synchronize Asana tasks within a workspace:
+
+```sh
+tw_asana_sync --tags asana --asana-workspace 123456789012345
```

Comment:
nitpick: Maybe we should allow the user to sync a workspace by specifying either its name or GID?

Reply:
We can do that. Unfortunately there's nothing prevent the user from having duplicate workspace names, but that's probably not a common situation. We can default to mapping name to GID, and if we see that there's ambiguity, exit and ask user to select GID?

Reply:
>  We can default to mapping name to GID, and if we see that there's ambiguity, exit and ask user to select GID?

Sounds like a good alternative.. but again, not too fussed about it if it's too much work

Reply:
Made changes in latest commit so that either (but not both) `--asana-task-gid` or `--asana-task-name` are accepted. If `--asana-task-name` is ambiguous, a `RuntimeError` is raised.